views/partials/head.ejs

```diff
@@ -0,0 +1,185 @@
+                0px 11px 15px rgb(0, 0, 0, 0.08);
+        }
+
+        .docs__section {
+            width: 60%;
+            margin: 1.5rem 0 1rem 0;
+            border-radius: 0.25rem;
+            background-color: #FBFAF5;
+            overflow: hidden;
+        }
+
+        .docs__content {
+            padding: 2rem;
+            height: 85vh;
+            overflow-y: scroll;
+            scroll-behavior: smooth;
+        }
+
+        .top__navbar {
+            width: 95vw;
+            max-width: 1440px;
+            margin: 0 auto;
+            height: 50px;
+            display: flex;
+            justify-content: space-around;
+            align-items: center;
+            border-bottom: 1px solid hsla(209, 75%, 50%, 8%);
+        }
+
+        .sidebar {
+            width: 20%;
+            padding: 1rem;
+            border-radius: 0.25rem;
+            background-color: #FBFAF5;
+            position: sticky;
+            top: calc(1rem + 60px);
+            height: 85vh;
+        }
+
+        .menu__item .menu__link {
+            text-decoration: none;
+            color: inherit;
+        }
+
+        hr {
+            border: 1px solid var(--clr-opaque-black-outline);
+        }
+
+        .sidebar details {
+            cursor: pointer;
+        }
+
+        li {
+            list-style-image: url(https://res.cloudinary.com/clouduser/image/upload/v1644944161/chevrons-right_domzra.svg);
+        }
+
+        li:hover {
+            text-decoration: underline hsla(209, 75%, 50%);
+        }
+
+        .component__container {
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: space-evenly;
+            align-items: center;
+        }
+
+        code {
+            border-radius: 0.25rem;
+        }
+
+        code span {
+            font-family: 'Fira Code', monospace;
+            font-size: 0.875rem;
+        }
+
+        .example-in-article {
+            background-color: var(--clr-primary-disabled);
+        }
+
+        .docs__content>*+* {
+            margin: 2rem 0 0 0;
+        }
+
+        @media (max-width: 1200px) {
+
+            .sidebar__two {
+                display: none;
+            }
+
+            .docs__section {
+                width: 75%;
+            }
+        }
+
+        @media (max-width: 768px) {
+
+            .main__container {
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+            }
+
+            .docs__section {
+                width: 100%;
+                margin: 0.5rem 0;
+            }
+
+            .docs__content {
+                padding: 1rem;
+                height: auto;
+                overflow: initial;
+            }
+
+            .sidebar__one {
+                width: 100%;
+                height: auto;
+                padding: 1rem;
+                margin-top: 1rem;
+                position: static;
+            }
+        }
+    </style>
+
```

Comment:
Also you can remove all these styling from here and create a seprate file for them